src/Attachment.php

```diff
@@ -125,6 +125,13 @@ class Attachment
      */
     protected $fields;
 
+    /**
+     * Message Formatting.
+     *
+     * @var array
+     */
+    protected $mrkdwn = [];
```

Comment:
Rename this to `useMarkdown`, let's store a boolean there with `false` as default

Reply:
Done.